src/contexts/tokens.tsx

```diff
@@ -31,8 +32,9 @@ const NEON_TOKEN_MODEL = {
 
 export function TokensProvider({ children = undefined }) {
   const { get } = useHttp()
-  const { chainId } = useNetworkType()
-  const neonChain = useMemo(() => chainId === CHAIN_IDS.devnet, [chainId])
+
+  const { connectedWallet, currentProvider } = useContext(WalletContext)
+  const neonChain = useMemo(() => currentProvider && Number(currentProvider.networkVersion) === CHAIN_IDS.devnet, [currentProvider])
```

Comment:
@Tkachez, you can't use `currentProvider.networkVersion`, not every of them will have networkVersion. You need to use `provider.request({ method: 'eth_chainId' })` to get provider chainId.